Tests/Tests.Maui/Tests.Maui.csproj

```diff
@@ -66,9 +70,39 @@
     </NativeReference>
   </ItemGroup>
 
+  <ItemGroup Condition="$(TargetFramework.Contains('-windows')) AND '$(UseRealmNupkgsWithVersion)' == ''">
+    <None Include="$(MSBuildProgramFiles32)\Windows Kits\10\bin\x64\ucrt\ucrtbased.dll" Condition="'$(Configuration)' == 'Debug'">
```

Comment:
Do we still need to include this?

Reply:
No, explicitly pulling in the Universal CRT debug library is no longer needed.